docs/section12/Rule12-5.md

```diff
@@ -0,0 +1,38 @@
+
+# Receptacle - Rule 12-5
+
+**Rule ID:** 12-5  
+**Rule Description:** User RMR Receptacle Power in Proposed RMR?  
+**Rule Assertion:** Proposed RMR = User RMR  
+**Appendix G Section:** Receptacle  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR and U_RMR  
+**Applicability Checks:**  
+
+  1. Rule 12-3 = True  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- **Applicability Check 1:** ```if (rule-12-3.status == True):```
+- Get the receptacle load of each space in the building segment in the User model: ```For building_segment in U_RMR.building.building_segments:```  
+  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
```

Comment:
Please use _user and _proposed suffixes for all variables to reduce confusion

Reply:
Updated in #26